### ISSUE TITLE
Build crashing

### ISSUE THREAD

**Issue by ForsakenHarmony** [Zodiase/meteor-mdl#43]
```
C:\Users\ArmyOfAnarchists\Programming\Web\WebstormProjects\datingwebsite>meteor build ..\build

C:\Users\ArmyOfAnarchists\AppData\Local\.meteor\packages\fourseven_scss\3.8.0_1\plugin.compileScssBatch.os.windows.x86_32\npm\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:165
      throw error;
            ^
Error: EPERM, symlink '..\acorn\bin\acorn'

C:\Users\ArmyOfAnarchists\Programming\Web\WebstormProjects\datingwebsite>
```

**Comment by Zodiase** [comment]
Looks like an issue of `fourseven_scss`?

**Comment by gbelot2003** [comment]
same with me, on a fresh install

npm ERR! Linux 4.5.7-202.fc23.x86_64
npm ERR! argv "node" "/home/gbelot/.meteor/packages/meteor-tool/.1.3.5_1.1sksnpz++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm" "rebuild" "--no-bin-links" "--update-binary"
npm ERR! node v0.10.46
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! thread-sleep@1.0.4 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 8
support request:
npm ERR!
/home/gbelot/.meteor/packages/fourseven_scss/.3.4.3.td7313++os.linux.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.linux.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/.temp-1uzkb3t/npm-debug.log

**Comment by mattkraemer** [comment]
Add Fourseven:SCSS (`meteor add fourseven:scss`) first and try again. That worked for me.

**Comment by Zodiase** [comment]
@gbelot2003 I think the error you encountered is different than the one OP is talking about. I haven't been able to reproduce OP's error but I can confirm the error you are experiencing is newly introduced in Meteor 1.3.5. I tried adding the package in every release of Meteor and upon hitting 1.3.5 I saw this error. Though I'm not sure what change caused this error.

@malmosands Thank you for providing the solution. I can confirm it works in Meteor 1.3.5.

FYI, if you try `meteor add fourseven:scss@=3.4.3` in Meteor 1.3.5, you would see that error, so there's something with `fourseven:scss@3.4.3` that is not working with Meteor 1.3.5. And `fourseven:scss@3.8.0_1` solves that.

I'll release a minor package version update to get rid of this error on installs.
